novawallet/Assets.xcassets/iconPencil.imageset/Contents.json

```diff
@@ -1,7 +1,7 @@
 {
   "images" : [
     {
-      "filename" : "iconPencil.pdf",
+      "filename" : "iconPencil.svg",
```

Comment:
Is there any pros we can get by using svg here instead of pdf?

Reply:
The problem was not the format, but the icon that was opaque so it was impossible to tint it. Updated to new version of asset in pdf format.